.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+name: CD - Publish package
+
+on:
+  release:
+    types: [created]
```

Comment:
```suggestion
    types: [released]
```

Reply:
Why this change? `created` is only triggered by non-draft releases, isn't this what we want?

Reply:
Not really.
> `created`: a draft is saved, or a release or pre-release is published without previously being saved as a draft

Reply:
Well that sounds reasonable to me, why is `released` better?

Reply:
Because with `released`, unlike others types, the workflow is only triggered on "stable" releases (published pre-release, or published draft).
Anyway, releases are normally only released by Release Please. But to be perfect and to be sure of the behavior in case a release draft or pre-release is made, the `released` type is the best.